.github/workflows/ci.yml

```diff
@@ -39,6 +38,29 @@ jobs:
       run: pip install -e .[full]
       shell: bash
 
-    - name: 'Run tests'
+    - name: 'Run test coverage statistics'
+      id: stats
+      shell: bash
       run: |
-        pytest -q tests
\ No newline at end of file
+        pip install pytest-cov
+        TEST_COVERAGE="$(pytest --cov-report term:skip-covered --cov=tiledb tests/ | grep '^TOTAL' | awk -v N=4 '{print $N}')"
+        echo "::set-output name=COVERAGE::$TEST_COVERAGE"
+        REF=${{ github.ref }}
+        echo "github.ref: $REF"
+        IFS='/' read -ra PATHS <<< "$REF"
+        BRANCH_NAME="${PATHS[1]}_${PATHS[2]}"
+        echo $BRANCH_NAME
+        echo $TEST_COVERAGE
+        echo "BRANCH=$(echo ${BRANCH_NAME})" >> $GITHUB_ENV
```

Comment:
Are all these `echo` for debugging or they're actually needed for setting the badge?

Reply:
Mainly debugging if someone needs to check the runner. I could remove them

Reply:
These should be removed, lets try to keep everything clean.